classes/class-install.php

```diff
@@ -71,12 +71,6 @@ public function __construct( $plugin ) {
 		$this->db_version = $this->get_db_version();
 		$this->stream_url = self_admin_url( $this->plugin->admin->admin_parent_page . '&page=' . $this->plugin->admin->settings_page_slug );
 
-		// Check DB and display an admin notice if there are tables missing.
-		add_action( 'init', array( $this, 'verify_db' ) );
```

Comment:
@kidunot89 I think removing this now makes the Travis checks fail because the table for the plugin is not created on the first run https://travis-ci.com/github/xwp/stream/jobs/468947527#L511